yarn-project/noir-contracts/src/scripts/copy_output.ts

```diff
@@ -40,27 +42,31 @@ function writeToProject(abi: any) {
 
 /**
  * Creates an Aztec function entry.
- * @param type - The type of the function.
- * @param params - The parameters of the function.
+ * @param type - The type of the function (secret | open | unconstrained).
+ * @param params - The parameters of the function ( name, type, visibility ).
  * @param returns - The return types of the function.
  * @param fn - The nargo function entry.
  * @returns The Aztec function entry.
  */
-function getFunction(type: FunctionType, params: ABIParameter[], returns: ABIType[], fn: any) {
-  if (!params) throw new Error(`ABI comment not found for function ${fn.name}`);
+function getFunction(type: FunctionType, params: ABIParameter[], returns: ABIType, fn: any): FunctionAbi {
+  // if (!params) throw new Error(`ABI comment not found for function ${fn.name}`);
```

Comment:
I think I commented this out, because I don't think the `/// ABI...` comments are needed anymore?

Reply:
Good catch, deleting it!